File: fizzbuzz.cpp
#include <iostream>
#include <string>


using namespace std;

int main() {
    for (int i = 1; i <= 30; i++) {
        string result = (i % 3 == 0) ? "fizz" : "";
        result += (i % 5 == 0) ? "buzz" : "";
        result += (result == "") ? to_string(i) : "";

        cout << result << endl;
    }
}

File: fizzbuzz.java
package com.company;

public class Main {

    public static void main(String[] args) {
        for (int i = 1; i <= 30; i++){

            String result = (i % 3 == 0) ? "fizz" : "";
            result += (i % 5 == 0) ? "buzz" : "";
            result += (result.length() == 0)? Integer.toString(i) : "";

            System.out.println(result);
        }
    }
}
